src/components/gamespec/Board.tsx

```diff
@@ -274,7 +306,7 @@ class Board extends React.Component {
                   onClick={() => {
                     this.handleClickOn(index);
                   }}
-                  item={item}
+                  // item={item}
```

Comment:
This should not be commented I think?

Reply:
Just wanted to clarify... is item={item} passing down all other props down to the child component, or is there an actual prop called "item"? Thanks!

Reply:
It is getting passed down, it is used somewhere, but I don't remember where.

Reply:
Got it, it is used by react-dnd when we drag and drop.